en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md

```diff
@@ -1,206 +1,162 @@
+The following sections describe how to set up Oracle RAC to replace the default H2 database in your WSO2 product:
 
-!!! info
-To test products on Oracle RAC, rename `oracle_rac.sql` to `oracle.sql` before running `-Dsetup` .
+-   [Setting up the database and users](#setting-up-the-database-and-users)
+-   [Setting up the drivers](#setting-up-the-drivers)
+-   [Executing db scripts to create tables on Oracle RAC database](#executing-db-scripts-to-create-tables-on-oracle-rac-database)
 
+Oracle Real Application Clusters (RAC) is an option that facilitates clustering and high availability in Oracle database environments. In the Oracle RAC environment, some of the commands used in `oracle.sql` are considered inefficient. Therefore, the product has a separate SQL script, `oracle_rac.sql` for Oracle RAC. The Oracle RAC-friendly script is located in the `dbscripts` folder together with other `.sql` scripts.
 
 ### Setting up the database and users
 
-Follow the steps below to set up an Oracle RAC database.
+Follow the steps below to set up an Oracle RAC database:
 
-1.  Set environment variables &lt; `ORACLE_HOME>` , `PATH` , `` and `ORACLE_SID` with the corresponding values ( `/oracle/app/oracle/product/11.2.0/dbhome_1` , `$PATH:<ORACLE_HOME>/bin` , and `orcl1` ) as follows:
-    ![]({{base_path}}/assets/attachments/126562524/126562530.png)2.  Connect to Oracle using SQL\*Plus as SYSDBA.
-    ![]({{base_path}}/assets/attachments/126562524/126562542.png)3.  Create a database user and grant privileges to the user as shown below:
+1.  As SYSDBA, create a database user and grant privileges to the user as shown below:
 
-    ``` powershell
-        Create user <USER_NAME> identified by password account unlock;
-        grant connect to <USER_NAME>;
-        grant create session, create table, create sequence, create trigger to <USER_NAME>;
-        alter user <USER_NAME> quota <SPACE_QUOTA_SIZE_IN_MEGABYTES> on '<TABLE_SPACE_NAME>';
-        commit;
+    ```sh
+    CREATE USER '<USER_NAME>' IDENTIFIED BY '<PASSWORD>' ACCOUNT UNLOCK;
+    GRANT CONNECT TO '<USER_NAME>';
+    GRANT CREATE SESSION, CREATE TABLE, CREATE SEQUENCE, CREATE TRIGGER, CREATE PROCEDURE TO '<USER_NAME>';
+    ALTER USER '<USER_NAME'> QUOTA '<SPACE_QUOTA_SIZE_IN_MEGABYTES>' ON '<TABLE_SPACE_NAME>';
+    GRANT DBA TO '<USER_NAME>';
+    COMMIT;
     ```
 
-4.  Exit from the SQL\*Plus session by executing the `quit` command.
+1.  Exit from the session by executing the `quit;` command.
 
-### Setting up the JDBC driver
+### Setting up the drivers
 
 Copy the Oracle JDBC libraries (for example, the `<ORACLE_HOME>/jdbc/lib/ojdbc14.jar` file) to the `<PRODUCT_HOME>/repository/components/lib/` directory.
 
-!!! info
-Remove the old database driver from the `<PRODUCT_HOME>/repository/components/dropins` directory when you upgrade the database driver.
-
+1. Unzip the WSO2 API Manager pack. Let's call it `<API-M_HOME>`.
```

Comment:
Shall we change as "call it as `<API-M_HOME>`"

Reply:
fixed with https://github.com/wso2/docs-apim/pull/619/commits/2c49dea90e2c24ab57b4dbf7c8549ff78be98a05